### DISCUSSION TITLE
Add ctrl+click hotkey to open links on windows

### DISCUSSION THREAD

**Discussion by dec0dOS** [outline/rich-markdown-editor#381] [Ideas] [answered]
The current version doesn't support standard CTRL+clicking behavior to open new links. Only meta key which is only acceptable on Mac. 
https://github.com/outline/rich-markdown-editor/blob/912e4693aab72a9bff5a1ef09e55eeb606cf75e3/src/marks/Link.ts#L119

**Comment by tommoor** [comment, marked as answer]
Fixed up in next release: https://github.com/outline/rich-markdown-editor/commit/67f57d907eb9bb733b2f405671c454588116c970

**Reply by dec0dOS** [reply]
Thanks!
